### PR TITLE
add debian 12 bookworm

### PR DESCRIPTION
**Pull request by unidevel**
Add debian-12.yaml, rename debian.yaml to debian-11.yaml, link debian.yaml to debian-12.yaml, and update docs.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ Distro:
 - [`archlinux.yaml`](./archlinux.yaml): ⭐Arch Linux
 - [`centos-stream-8.yaml`](./centos-stream-8.yaml): CentOS Stream 8
 - [`centos-stream-9.yaml`](./centos-stream-9.yaml), `centos-stream.yaml`: CentOS Stream 9
-- [`debian.yaml`](./debian.yaml): ⭐Debian GNU/Linux
+- [`debian.yaml`](./debian-11.yaml): ⭐Debian GNU/Linux 11(bullseye)
+- [`debian.yaml`](./debian-12.yaml): ⭐Debian GNU/Linux 12(bookworm)
```
Comment (AkihiroSuda) [inline examples/README.md:15]:
```suggestion
- [`debian-12.yaml`](./debian-12.yaml), `debian.yaml`: ⭐Debian GNU/Linux 12(bookworm)
```

File: examples/README.md [hunk 2]
```diff
@@ -11,7 +11,8 @@ Distro:
 - [`archlinux.yaml`](./archlinux.yaml): ⭐Arch Linux
 - [`centos-stream-8.yaml`](./centos-stream-8.yaml): CentOS Stream 8
 - [`centos-stream-9.yaml`](./centos-stream-9.yaml), `centos-stream.yaml`: CentOS Stream 9
-- [`debian.yaml`](./debian.yaml): ⭐Debian GNU/Linux
+- [`debian.yaml`](./debian-11.yaml): ⭐Debian GNU/Linux 11(bullseye)
```
Comment (AkihiroSuda) [inline examples/README.md:14]:
```suggestion
- [`debian-11.yaml`](./debian-11.yaml): Debian GNU/Linux 11(bullseye)
```

File: examples/debian-12.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# This example requires Lima v0.7.0 or later
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+- location: "https://cloud.debian.org/images/cloud/bullseye/20230515-1381/debian-11-genericcloud-amd64-20230515-1381.qcow2"
```
Comment (AkihiroSuda) [inline examples/debian-12.yaml:4]: Seems debian 11

Comment (unidevel) [reply examples/debian-12.yaml]: Thanks, I copied wrong file by mistake.

File: examples/README.md (hunk 1 above)
Comment (unidevel) [reply examples/README.md]: updated

File: examples/README.md (hunk 2 above)
Comment (unidevel) [reply examples/README.md]: updated

File: examples/README.md [hunk 4]
```diff
@@ -11,7 +11,8 @@ Distro:
 - [`archlinux.yaml`](./archlinux.yaml): ⭐Arch Linux
 - [`centos-stream-8.yaml`](./centos-stream-8.yaml): CentOS Stream 8
 - [`centos-stream-9.yaml`](./centos-stream-9.yaml), `centos-stream.yaml`: CentOS Stream 9
-- [`debian.yaml`](./debian.yaml): ⭐Debian GNU/Linux
+- [`debian-11.yaml`](./debian-11.yaml): ⭐Debian GNU/Linux 11(bullseye)
```
Comment (AkihiroSuda) [inline examples/README.md:14]:
```suggestion
- [`debian-11.yaml`](./debian-11.yaml): Debian GNU/Linux 11(bullseye)
```

Comment (unidevel) [reply examples/README.md]: updated

File: hack/test-example.sh [hunk 5]
```diff
@@ -217,7 +217,7 @@ if [[ -n ${CHECKS["port-forwards"]} ]]; then
 		limactl shell "$NAME" sudo pacman -Syu --noconfirm openbsd-netcat
 	fi
 	if [ "${NAME}" = "debian" ]; then
-		limactl shell "$NAME" sudo apt-get install -y netcat
+		limactl shell "$NAME" sudo apt-get install -y netcat-openbsd
```
Comment (unidevel) [inline hack/test-example.sh:220]: I tested this with debian 11 and debian 12, both works.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**